test/transforms.jl

```diff
@@ -41,4 +41,16 @@
     @test nelements(smesh) == nelements(mesh)
     @test topology(smesh) == topology(mesh)
   end
+
+  @testset "Rotation" begin
+    # a rotation doesn't change the topology
+    trans = Rotation([1, 1, 1], 45)
+    @test TB.isrevertible(trans)
+    mesh  = readply(T, joinpath(datadir,"beethoven.ply"))
+    smesh = trans(mesh)
+    @test nvertices(smesh) == nvertices(mesh)
+    @test nelements(smesh) == nelements(mesh)
+    @test topology(smesh) == topology(mesh)
```

Comment:
We need other basic tests with trivial geometries to check that the coordinates have been correctly rotated. For example, you can create a simple `Triangle((0,0), (1,0), (0,1))` and apply the rotation to it. Then check that the vertices satisfy some condition.